namesys/base.go

```diff
@@ -25,15 +25,6 @@ func resolve(ctx context.Context, r resolver, name string, options *opts.Resolve
 		}
 		log.Debugf("resolved %s to %s", name, p.String())
 
-		if strings.HasPrefix(p.String(), "/ipfs/") {
```

Comment:
I've changed this check to "is not /ipns/". That way, we continue resolving until we can no longer resolve.